app/controllers/student_applications_controller.rb

```diff
@@ -11,6 +11,7 @@ def create
     @student_application.semester = Semester.current
     if @student_application.save_and_update_user
       flash[:info] = "Thanks for applying! We'll get back to you with our decision."
```

Comment:
Maybe we can say "Thanks for applying! Please check your email for a confirmation." since they know we'll get back to 'em haha